vee-mainnet.conf

```diff
@@ -0,0 +1,66 @@
+# Vee node settings
+vee {
+  # Application logging level. Could be DEBUG | INFO | WARN | ERROR. Default value is INFO.
+  logging-level = INFO
+
+  # P2P Network settings
+  network {
+    known-peers = []
+
+    # Network address
+    bind-address = "0.0.0.0"
+
+    # Port number
+    port = 9921
+
+    # Node name to send during handshake. Comment this string out to set random node name.
+    # node-name = "My MAINNET node"
+
+    # String with IP address and port to send as external address during handshake. Could be set automatically if uPnP is enabled.
+    # declared-address = "1.2.3.4:9921"
+  }
+
+  # Wallet settings
+  wallet {
+    # Password to protect wallet file
+    password = "ridethevee!"
```

Comment:
maybe it is too late, but we could have changed this to "virtual economy era!" or something :)